.gitignore

```
@@ -4,19 +4,18 @@ tests/data/*.json
 tests/pdal/test_pdalc_*.c
 !tests/pdal/test_pdalc_utils.c
 
-# Ignore doxygen docs
-/doc/doxygen
-/doc/Doxyfile
-
 # Select VS Code configurations
 .vscode/ipch/**
 .vscode/settings.json
 .vscode/c_cpp_properties.json
 
+.vs/
+
 # Vim ancillary files
 *~
 *.swp
 
+
```

I recommend ignoring Mac's .DS_Store file:

```suggestion
.DS_Store
```